developer-extension/src/panel/hooks/useEvents.ts

```diff
@@ -114,3 +136,11 @@ function listenRequests(callback: (events: StoredEvent[]) => void) {
 
   return () => chrome.devtools.network.onRequestFinished.removeListener(beforeRequestHandler)
 }
+
+function listenEventsFromSdk(events: (events: StoredEvent[]) => void) {
+  return listenSdkMessages((message) => {
+    if (message.type === 'logs' || message.type === 'rum' || message.type === 'telemetry') {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏are you planning to handle records in another PR?